picard/ui/playertoolbar.py

```diff
@@ -99,6 +99,8 @@ def create_toolbar(self):
 
     def set_objects(self, objects):
         self._selected_objects = objects
+        player_enabled = len(objects) > 0
```

Comment:
Not strictly needed.

```python
self._toolbar.play_action.setEnabled(self._selected_objects)
```

Reply:
Fixed as part of https://github.com/metabrainz/picard/pull/1231/commits/164c32ff0131de30f5a97634f1a97e5de157f46c